tests/test_browser.py

```diff
@@ -2408,6 +2408,16 @@ def test_preload_module(self):
           REPORT_RESULT(3);
           return 3;
         }
+        void *invalid_handle = dlopen("/invalid.so", 0);
+        if (invalid_handle) {
+          REPORT_RESULT(4);
+          return 4;
+        }
+        void *invalid_handle2 = dlopen("/invalid.so", 0);
+        if (invalid_handle2) {
+          REPORT_RESULT(5);
+          return 5;
+        }
```

Comment:
Can you make this a separate test?    It doesn't seem related to preloading, right?